eng/helix/content/runtests.sh

```diff
@@ -49,15 +52,15 @@ if [[ -z "${10:-}" ]]; then
     }
 else
     echo "InstallDotNet $DOTNET_ROOT $dotnet_sdk_version '' '' true https://dotnetclimsrc.blob.core.windows.net/dotnet ..."
-    InstallDotNet $DOTNET_ROOT $dotnet_sdk_version "" "" true https://dotnetclimsrc.blob.core.windows.net/dotnet ${10} || {
+    InstallDotNet $DOTNET_ROOT $dotnet_sdk_version "" "" true https://dotnetclimsrc.blob.core.windows.net/dotnet ${11} || {
```

Comment:
This doesn't align with [line 38](https://github.com/dotnet/aspnetcore/pull/30113/files#diff-408e316c77dc35895679b7dd766f75c9a0a2d0afaad1c9ca97e9153730bff3a6R38). Must change that line too unless you _want_ internal PRs to fail😺

Separately, I don't see any real use of `$(10}` in this script. Shouldn't that be used to control placing `InstallPlaywright ` in the environment before running `RunTests/RunTests.csproj`❔

Reply:
Thanks, I'll fix line 38, yeah the next PR focuses on getting things working on linux/mac, but it doesn't really matter for this one, see https://github.com/dotnet/aspnetcore/pull/30404/checks?check_run_id=2017759237

Reply:
> it doesn't really matter for this one

What allows Linux and macOS jobs to work e.g. are the Playwright tests disabled there❔

Reply:
These are WindowsOnlyTests right now so yeah they aren't sent to those queues at all